dataactcore/models/domainModels.py

```diff
@@ -85,7 +85,7 @@ class SF133(Base):
     ending_period_of_availabil = Column(Text)
     main_account_code = Column(Text, nullable=False)
     sub_account_code = Column(Text, nullable=False)
-    tas = Column(Text, nullable=False, default=concatTas, onupdate=concatTas)
```

Comment:
Are we not concatenating on update because we now have a `tas_id` associated with each entry?